Moving emergency phone to a new tagging scheme

kocio-pl opened the pull request:
Resolves #318.

Name selection in SQL was removed because we're not displaying names for such POIs.

pnorman commented on project.mml:
```diff
@@ -1525,6 +1528,7 @@ Layer:
             OR "natural" IN ('peak', 'volcano', 'saddle', 'spring', 'cave_entrance')
             OR historic IN ('memorial', 'monument', 'archaeological_site', 'wayside_cross')
             OR tags->'memorial' IN ('plaque')
+            OR tags->'emergency' IN ('phone')
```
This [should be `tags @> 'emergency=>phone'`](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#sql-style-guidelines), as well as the repetition in the WHERE clause for the other layer.

kocio-pl replied:
Hm, but why should we check if a tag is in the hstore? Why do we need this special construction at all?

pnorman replied:
If we don't include an item in the OR part of the WHERE clause it won't be selected in the layer.

kocio-pl replied:
The icon on a node was present in my test rendering, so it looks for me that was selected.

pnorman replied:
The problem is `OR tags->'emergency' IN ('phone')` does not follow our style standards.

kocio-pl replied:
I see. Does `OR tags->'memorial' IN ('plaque')` have this problem too?

pnorman replied:
Yes, but you don't need to change it in this PR, it's an existing error.

kocio-pl commented:
I have changed SQL formatting as requested. 

I have also removed support for rendering such polygons, because there are only 34 of them, Wiki definition does not allow areas and they all look like mistagging (property of building or part of a road).
